src/Enhavo/Bundle/SliderBundle/Resources/views/Widget/slider.html.twig

```diff
@@ -1,12 +1,18 @@
 {% block slider %}
-<ul class="slider">
+<ul class="slider teeeest">
     {% for slide in slides %}
         {% if slide.image %}
             <li class="slider-img" style="background-image:url({{ media_url(slide.image) }}); background-size:cover; background-attachment: fixed;">
+                {% if slide.url is not null %}
+                    <a href="http://{{ slide.url }}" class="slide-link">
```

Comment:
Dont'use http as prefix, that should be done in cms or we use some function here. Imaging a user uses http in the backend input